src/hal/components/gantry.comp

```diff
@@ -74,6 +74,7 @@ license "GPL";
 variable float offset[7] = 0.0;
 variable float prev_cmd = 0.0;
 variable int   fb_joint = 0;
+variable inti  latching = 0;
```

Comment:
On 1/14/2015 10:22 AM, John Morris wrote:
>> @@ -74,6 +74,7 @@ license "GPL";
>>  variable float offset[7] = 0.0;
>>  variable float prev_cmd = 0.0;
>>  variable int   fb_joint = 0;
>> +variable inti  latching = 0;
> 
> Oops, typo!

Yeah, fixed and force-pushed.  I'm not sure if that will restart the
buildbot or not.

...yep, it's chewing on it again.

-- 
Charles Steinkuehler
charles@steinkuehler.net